input/input.css

```diff
@@ -0,0 +1,30 @@
+@import '../../utility/common.css';
+@import '../../installation/styles.css';
+
+.input--container {
+  width: 20em;
+}
+.lbl {
+  font-size: 1em;
+  color: var(--heading);
+  margin: 0 0.5em;
+}
+.ipt {
```

Comment:
could have used full name as input rather than ipt as it becomes more understandable